packages/shared/styles/components/organisms/SfContentPages.scss

```diff
@@ -1,95 +1,85 @@
 @import "../../helpers";
 
 .sf-content-pages {
+  height: calc(100vh - 3.125rem);
   overflow: hidden;
-  height: 100vh;
+  .sf-bar {
+    position: var(--content-pages-bar-position, fixed);
+    --bar-height: 3.125rem;
+    width: 100%;
+  }
   &__section {
-    box-sizing: border-box;
     display: flex;
     height: 100%;
+    margin: var(--content-pages-section-margin, var(--spacer-2xl) 0 0 0);
     transition: var(--content-pages-transition, transform 150ms ease-in-out);
     &--is-active {
-      transform: translateX(-100%);
+      transform: var(
+        --content-pages-section-active-transform,
+        translateX(-100%)
+      );
     }
   }
-  &__sidebar,
-  &__content {
-    overflow-y: auto;
-    box-sizing: border-box;
-    height: calc(100vh - 5rem);
-  }
   &__sidebar {
     flex: var(--content-pages-sidebar-flex, 0 0 100%);
-    padding: var(--content-pages-sidebar-padding, var(--spacer-big));
-    background-color: var(--content-pages-sidebar-background);
-  }
-  &__content {
-    flex: 0 0 100%;
-    padding: var(--content-pages-content-padding, var(--spacer-big));
+    padding: var(--content-pages-sidebar-padding);
+    background: var(--content-pages-sidebar-background);
+    overflow-y: auto;
   }
   &__title {
-    padding: var(--content-pages-title-padding, var(--spacer-big) 0);
-    margin: var(--content-pages-title-margin);
-    font: var(
-      --content-pages-title-font,
-      font(
-        --content-pages-title,
-        500,
-        var(--font-size-big),
-        1.6,
-        var(--font-family-secondary)
-      )
+    margin: var(--content-pages-sidebar-title-margin, 0 0 0 var(--spacer-base));
+    @include font(
```

Comment:
Can you please replace all the `<h1> <h2>` with `SfHeading ` with a proper `level` prop